docs/blog/v5-getting-started.md

```diff
@@ -0,0 +1,313 @@
+# Version 5 getting started
```

Comment:
You won't be able to follow the getting started guide since we haven't done any release of version 5. 
So no helm/container image have been pushed to a central repo.
But hopefully you get the picture any way.